contracts/cairo/.env.example

```diff
@@ -0,0 +1,16 @@
+# To deploy Starknet contract 
+
+## Starkli
+STARKNET_ACCOUNT=<STARKNET_ACCOUNT_SRC>
+STARKNET_KEYSTORE=<STARKNET_ACCOUNT_SRC>
+STARKNET_RPC=<STARKNET_RPC>
+
+## Required envs
+HERODOTUS_FACTS_REGISTRY=0x01b2111317EB693c3EE46633edd45A4876db14A3a53ACDBf4E5166976d8e869d
+MM_ETHEREUM_WALLET=0xE8504996d2e25735FA88B3A0a03B4ceD2d3086CC
+ETH_CONTRACT_ADDR=0xdd69db25f6d620a7bad3023c5d32761d353d3de9
+NATIVE_TOKEN_ETH_STARKNET=0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
+SN_WALLET_ADDR=<SN_WALLET_ADDR>
+
+# To upgrade Starknet contract 
+SN_PROXY_ESCROW_ADDRESS=<SN_PROXY_ESCROW_ADDRESS>
```

Comment:
Can't seem to understand where should i get this value. Could you add it to the Readme?

Reply:
Sure! :+1: